app/internal_packages/composer-templates/lib/template-store.ts

```diff
@@ -30,7 +30,7 @@ class TemplateStore extends MailspringStore {
     this.listenTo(Actions.deleteTemplate, this._onDeleteTemplate);
     this.listenTo(Actions.renameTemplate, this._onRenameTemplate);
 
-    const welcomeName = 'Welcome to Templates.html';
+    const welcomeName = localized('Welcome to Templates') + '.html';
     const welcomePath = path.join(__dirname, '..', 'assets', welcomeName);
```

Comment:
Hmm I think we may need to support falling back to the english version here if the localized version doesn't exist. This is a nice feature though!